<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetNfeEntrada</class>
 <widget class="QWidget" name="WidgetNfeEntrada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="TableView" name="tableResumo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Filtros</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxStatus">
          <property name="title">
           <string>Status</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_40">
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBoxPendente">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Resumo</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxAutorizado">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Autorizada</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBoxCancelado">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Cancelada</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBoxUtilizada">
          <property name="title">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_42">
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBoxInutilizada">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Inutilizada</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxUtilizada">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Utilizada</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBoxMes">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Data</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateEditAte">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="dateEditDe">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>De</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>At√©</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBoxLojas">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Loja</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="ItemBox" name="itemBoxLoja">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="TableView" name="table">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonInutilizarNFe">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Inutilizar NF-e</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExportar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Exportar XML/PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExportarExcel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Exportar Excel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFollowup">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Followup</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelBuscar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Buscar:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="lineEditBusca">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Emitente/NF-e/OC/Venda</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemBox</class>
   <extends>QLineEdit</extends>
   <header>src/itembox.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>groupBoxStatus</tabstop>
  <tabstop>checkBoxAutorizado</tabstop>
  <tabstop>checkBoxPendente</tabstop>
  <tabstop>checkBoxCancelado</tabstop>
  <tabstop>groupBoxLojas</tabstop>
  <tabstop>itemBoxLoja</tabstop>
  <tabstop>table</tabstop>
  <tabstop>pushButtonInutilizarNFe</tabstop>
  <tabstop>pushButtonExportar</tabstop>
  <tabstop>lineEditBusca</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
